_emails/oakland.md

```diff
@@ -1,41 +1,39 @@
 ---
 title: Oakland, CA
-permalink: "/oakland"
+permalink: "/defundopd"
 name: Letter to Mayor, Council Members, and Elected Officials
 city: Oakland
 state: CA
 recipients:
-- atlarge@oaklandca.gov
 - officeofthemayor@oaklandnet.com
 - dkalb@oaklandca.gov
 - nfbas@oaklandca.gov
-- LMcElhaney@oaklandca.gov
+- lmcelhaney@oaklandca.gov
 - district4@oaklandca.gov
-- Ngallo@oaklandca.gov
-- District6@oaklandca.gov
+- ngallo@oaklandca.gov
+- district6@oaklandca.gov
 - lreid@oaklandnet.com
-- ebernstein@oaklandcityattorney.org
-- abenson@oaklandca.gov
-- bjohnson@oaklandca.gov
-- askrcd-da@acgov.org
-subject: Oakland Resident for Defunding the OPD
+- atlarge@oaklandca.gov
+subject: Defund OPD.  Invest in Community.
 body: |-
-  To Mayor Schaaf, the Oakland City Council and Oakland and Alameda County Elected Officials:
+  Dear Mayor Schaaf and City Council Members,
 
-  My name is \[YOUR NAME\], and I am a resident of Oakland. This past week, our nation has been gripped by protests calling for a rapid and meaningful reconsideration of the role of policing in communities as well as an end to racism and anti-Blackness in America. Our city has been at the forefront of much of this action. Accordingly, it has come to my attention that the budget for 2021 is being decided as these protests continue.
+  OPD takes an enormous share of the city’s general fund, and that percentage has risen exponentially for the last two decades, taking away desperately needed resources from essential city programs and services. OPD has overspent its authorized overtime budget by 100% -- tens of millions of dollars -- in recent years, as individual cops bring home annual compensation upwards of $400k. The investment in policing has not made us safer -- OPD remains an embarrassment to the city and a lethal threat to Oakland’s Black and Brown communities, while increased police spending shows no correlation to decreasing crime levels over the past 20 years. With Oakland's current finances in dire jeopardy, it is clear that we must defund the police.
```

Comment:
It might still be worthwhile to include the `My name is \[YOUR NAME\], and I am a resident of Oakland.`

Reply:
cool, give me just a sec and I'll revise!